addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: AllPlay Binding
+Bundle-SymbolicName: org.openhab.binding.allplay;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/tchaikovsky.jar
+Bundle-NativeCode: lib/x64/win/alljoyn_java.dll;processor=amd64;osname=win32;osname=Windows8;osname=Windows 8;osname=Win8;osname=Windows10;osname=Windows 10;osname=Win10,
+ lib/x86/win/alljoyn_java.dll;processor=x86;osname=win32;osname=Windows8;osname=Windows 8;osname=Win8;osname=Windows10;osname=Windows 10;osname=Win10,  
+ lib/x86/linux/liballjoyn_java.so;processor=x86;osname=linux,
+ lib/x86-64/linux/liballjoyn_java.so;processor=amd64;osname=linux,
+ lib/arm/linux/liballjoyn_java.so;processor=arm;osname=linux,*
+Import-Package: com.google.common.collect,
+ org.apache.commons.io;version="2.2.0",
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.osgi.framework,
+ org.osgi.service.component,
+ org.slf4j
+Service-Component: OSGI-INF/*
+Export-Package: org.openhab.binding.allplay,
+ org.openhab.binding.allplay.handler
+Require-Bundle: org.eclipse.smarthome.core
```

Comment:
if everything compiles, you can remove it. If something is missing, add missing packages to Import-Package directive instead.